packages/main/src/components/Toolbar/index.tsx

```diff
@@ -72,9 +74,16 @@ const Toolbar: FC<ToolbarProptypes> = forwardRef((props: ToolbarProptypes, ref:
   const childrenWithRef = useMemo(() => {
     controlMetaData.current = [];
 
-    return React.Children.toArray(
-      (children as ReactElement)?.type === React.Fragment ? (children as ReactElement).props.children : children
-    ).map((item: ReactElement, index) => {
+    const refactoredChildren = React.Children.toArray(children).map((child, index) => {
+      if ((child as ReactElement).type === React.Fragment) {
+        return (child as ReactElement).props.children.map((item, itemIndex: number) => {
```

Comment:
Please filter out falsy children like `undefined` or `false`. Otherwise something like this will throw an error:
```
      <Toolbar>
        <>
          <Text>Hi</Text>
          {show && <Text>This will throw when show is false</Text>}
        </>
      </Toolbar>
```
```suggestion
        return (child as ReactElement).props.children.filter(Boolean).map((item, itemIndex: number) => {
```